Draft/new delivery docs (#6)
norsk-gjeldsinformasjon/api-docs: docs/howto/1. Select an integration.md

# Select an integration

There are a few different options for integrating with Norsk Gjeldsinformasjon. Skip to the heading that suits your need.

## We are a Financial Institution (credit bureaus or lender)

Which companies and institutions that can access information about unsecured debt is regulated in gjeldsinformasjonsloven. If your organization is allowed to access this information you can integrate with the Search API.

!!! note "Note"

>     Documentation for this service is being migrated. See the new - [Look up debt without consent](4.%20Lookup%20debt%20without%20consent.md) section for updated documentation.

Reviewer: ```suggestion
Documentation for that service can be found here - [Look up debt without consent](4.%20Lookup%20debt%20without%20consent.md)
```
